frameworks/g-cloud-7/messages/dates.yml

```diff
@@ -1,4 +1,5 @@
-clarifications_close_date: "5pm, 22 September 2015"
-clarifications_publish_date: "5pm, 29 September 2015"
-framework_close_date: "3pm, 6 October 2015"
-framework_live_date: "23 November 2015"
+clarifications_close_date: '5pm <abbr title="Greenwich Mean Time">GMT</abbr>, 22 September 2015'
```

Comment:
But that time isn't right then. We've closed submission on 3pm, when UK was on BST. So it's either 2pm UTC in October, or 3pm BST, since that's what the text said when G-7 was open.